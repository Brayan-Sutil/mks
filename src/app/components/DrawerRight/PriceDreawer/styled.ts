import { Box, Typography } from '@mui/material';
import styled from 'styled-components';

export const StyledBox = styled(Box)`
  display: flex !important;
  justify-content: space-between;
  position: absolute;
  bottom: 60px;
  right: 45px;
  width: 430px;
`;

export const StyledTypography = styled(Typography)`
    font-weight: bold !important;
    color: white !important;
    font-size: 30px !important;
`;
