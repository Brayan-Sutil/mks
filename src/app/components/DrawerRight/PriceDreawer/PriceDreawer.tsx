import { StyledBox, StyledTypography } from "./styled";

const PriceDreawer = () => {
    return (
      <StyledBox>
        <StyledTypography>Total:</StyledTypography>
        <StyledTypography>R$ {"798"}</StyledTypography>
      </StyledBox>
    );
}

export default PriceDreawer