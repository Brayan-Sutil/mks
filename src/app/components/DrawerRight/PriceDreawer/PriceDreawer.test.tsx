import { render } from "@testing-library/react";
import PriceDreawer from "./PriceDreawer";

describe("PriceDreawer", () => {
    it("should render without errors", () => {
        const { getByTestId } = render(<PriceDreawer />);
        expect(getByTestId("price-drawer")).toBeInTheDocument();
    });

    it("should render the correct title", () => {
        const { getByText } = render(<PriceDreawer />);
        expect(getByText("PreÃ§o")).toBeInTheDocument();
    });

    it("should render the correct price", () => {
        const { getByText } = render(<PriceDreawer />);
        expect(getByText("R$ 0,00")).toBeInTheDocument();
    });

    it("should match the snapshot", () => {
        const { container } = render(<PriceDreawer />);
        expect(container).toMatchSnapshot();
    });
});
