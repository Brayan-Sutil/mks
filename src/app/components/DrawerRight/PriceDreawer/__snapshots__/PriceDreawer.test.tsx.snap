// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceDreawer should match the snapshot 1`] = `
<div>
  <div
    class="sc-bdfCDU laxfuQ MuiBox-root css-0"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 sc-gsTDqH hPFdnW css-ahj2mt-MuiTypography-root"
    >
      Total:
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 sc-gsTDqH hPFdnW css-ahj2mt-MuiTypography-root"
    >
      R$ 
      798
    </p>
  </div>
</div>
`;
